PYME/IO/image.py

```diff
@@ -643,6 +643,65 @@ def _load_supertile(self, filename):
         self.mdh = data.mdh
         self.seriesName = filename
         self.mode = 'default'
+
+    def _load_concatenated(self, filename):
+        from PYME.IO.DataSources import ConcatenatedDataSource
+        import glob
+
+        if filename.upper().startswith('CONCATENATED://'):
+            filename = filename[15:]
+
+        z, t, c = None, None, None
+        if filename[0] == ':':
+            # we are explicitly passing ztc dimensions as ?z=zval&t=tval&c=cval
+            split_string = filename[1:].split(':')
+            dim_string = split_string[0]
+            z, t, c = [int(s) for s in dim_string.split(',')]
+            filename = ''.join(split_string[1:])
+
+        if '?' in filename:
+            #we have a query string to pick the series
+            from six.moves import urllib
+            filename, query = filename.split('?')
+            
+            try:
+                z = int(urllib.parse.parse_qs(query)['z'][0])
+            except KeyError:
+                z = 1
+            try:
+                t = int(urllib.parse.parse_qs(query)['t'][0])
+            except KeyError:
+                t = 1
+            try:
+                c = int(urllib.parse.parse_qs(query)['c'][0])
+            except KeyError:
+                c = 1
+
+            if ((z==1) and (t==1) and (c==1)):
+                #we have a query string but no dimensions specified
+                z, t, c = None, None, None
+
+        if filename.find(',') == -1:
```

Comment:
might need to revisit commas as a separator, but happy to deal with this when the time comes.